Makefile

```diff
@@ -434,8 +451,9 @@ RUNTIME_CPP_COMPONENTS = \
   posix_math \
   posix_print \
   posix_thread_pool \
-  to_string \
+  rs \
```

Comment:
I'm fine with renaming, but then yes - consistency is good. :-)
"rs" came from Android source code base(as in frameworks/rs).